contracts/svc/invocable-scripts.yml

```diff
@@ -278,7 +278,7 @@ components:
           type: string
     ScriptHTTPResponseData:
       description: The data sent to end user when a script is invoked using http. User defined and dynamic
-      type: object
+      type: string
```

Comment:
remember to not manually edit files in `/contracts` but instead update the file in  `/src` and use `make generate-all` to generate the contract